### PR TITLE
feat: create modal add user admin

### PR DESCRIPTION
**Pull request by yoslie**
## Changes
Create modal form add user admin in menu list master data user admin
image

## Evidence
title: Create modal form add user admin in menu list master data user admin
project: Desa Digital
participants: @yoslie @maulanayuseph @adzharamrullah @Ibwedagama @doohanas

### CODE REVIEW

File: components/Admin/Add/index.vue [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+      </div>
+      <div class="form-add-admin__form-group">
+        <BaseInputText
+          v-model="form.password"
+          name="password"
+          label="Kata Sandi"
+          placeholder="Masukkan kata sandi"
+          type="password"
+        />
+        <!-- <jds-input-text
```
Comment (doohanas) [inline components/Admin/Add/index.vue]: Minor,
Hello kak @yoslie, is this code still use?

**Comment by adzharamrullah** [comment]
recording

Hello @yoslie i think you should preview image on avatar box after user select picture from local computer


<details><summary>Browser metadata</summary>





```
Path:      /data-user
Browser:   Chrome 99.0.4844.51 on Mac OS 10.15.7
Viewport:  1440 x 679 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-27--cms-desadigital.netlify.app%2Fdata-user&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=99.0&resolution=1440x679&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-27--cms-desadigital.netlify.app/data-user) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6232b4a4a9925c000a99311c&commentID=6232ce0b65c6521cc6dcbd5d&resolution=resolved)

**Review by maulanayuseph — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

**Comment by yoslie** [comment]
> recording
> 
> Hello @yoslie i think you should preview image on avatar box after user select picture from local computer
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-27--cms-desadigital.netlify.app/data-user) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6232b4a4a9925c000a99311c&commentID=6232ce0b65c6521cc6dcbd5d&resolution=resolved)

I did'nt check the acceptance criteria, my bad lol, thanks for remind me..
updated [here](https://github.com/jabardigitalservice/dedi-cms/pull/27/commits/5341baeb87b8506753ea7539c730f13028c5d779) thanks @adzharamrullah 
preview:
telegram-cloud-photo-size-5-6168160717887352808-y

File: components/Admin/Add/index.vue (hunk 1 above)
Comment (yoslie) [reply components/Admin/Add/index.vue]: it's not used thanks @doohanas , fixed [here](https://github.com/jabardigitalservice/dedi-cms/pull/27/commits/5c9cfc24749bfa64728bfcc01f0a892ec21d68d2)

**Review by doohanas — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**